### PR TITLE
Fix bugs raised in PE-D

### PR DESCRIPTION
**Pull request by sun-ruiheng**
Fixed the bugs explicitly assigned to me so far.

Might add more commits for further bugs.

### CODE REVIEW

**Comment by sun-ruiheng** [comment]
Sorry for laziness earlier and not specifying the issues I targeted, but it's all the issues assigned to me that have been resolved. You can filter the issue list and get a comprehensive view

File: src/main/java/seedu/address/logic/parser/ListFavouriteCommandParser.java [hunk 1]
```diff
@@ -2,22 +2,22 @@
 
 import static seedu.address.logic.parser.ParserUtil.parseNoArgs;
 
-import seedu.address.logic.commands.ShowFavouriteCommand;
+import seedu.address.logic.commands.ListFavouriteCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
  * Parses input arguments and creates a new ShowFavouriteCommand object
```
Comment (guanquann) [inline src/main/java/seedu/address/logic/parser/ListFavouriteCommandParser.java:9]: Documentation not changed to new commands yet

**Review by gavingoh99 — COMMENTED** [review]
Did you update the DG to reflect `List favourite feature` instead of `Show favourite feature`

Referring to this part:
Screenshot 2024-04-12 at 12 03 45 AM

**Review by gavingoh99 — APPROVED** [review]
LGTM on the documentation changes!

**Pull request merged**